### PR TITLE
fix: code block color to improve contrast

### LINKED ISSUE #37153: Color contrast between codeblock and background negligible
The color contrast between the background and the code block is almost negligible. I believe the original intent of having different colors is that users can differentiate between the code example and the normal text.

image

### PR DESCRIPTION
**Pull request by DerrykBoyd**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37153

### CODE REVIEW

**Comment by raisedadead** [comment]
Hey @dboydgit thanks for the pull-request, do you mind posting a screenshot in here, in the comments for a quick look for us?

**Comment by dboydgit** [comment]
Sure, here is what the code blocks look like after the updates

image

File: client/src/components/layouts/prism.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
 code .token.operator {
   background: none;
 }
+
+:not(pre) > code[class*='language-'],
+pre[class*='language-'] {
+  background: var(--quaternary-background);
```
Comment (raisedadead) [inline client/src/components/layouts/prism.css]:
```suggestion
  background: var(--primary-background);
}

* {
  text-shadow: none !important;
}
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Review by raisedadead — CHANGES_REQUESTED** [review]
Also in `globals.css` I would prefer to add a `line-height` of `1.5rem` on the `p` tags

**Comment by dboydgit** [comment]
I've made those updates, layout now looks like

image

**Comment by raisedadead** [comment]
You may need to run a check on the linting.

**Comment by dboydgit** [comment]
I think there was an errant `}` in the first commit. Second commit should be OK.

**Comment by raisedadead** [comment]
> I think there was an errant `}` in the first commit. Second commit should be OK.

Great thanks for confirming and fixing it :)

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks @dboydgit, for this pull request and contributing to the code-base for the first time.
We will be looking forward for more contributions in future.

Cheers and happy contributing!

**Pull request merged**